### PR TITLE
feat(website): implement the responsive design

### LINKED ISSUE #42: implement mobile designs into website

### PR DESCRIPTION
**Pull request by LorenzoValentine**
close #42

### CODE REVIEW

File: src/components/NavBar.tsx [hunk 1]
```diff
@@ -12,43 +12,47 @@ interface NavBarProps {
 const Header = styled.header`
   display: flex;
   align-items: center;
-  justify-content: space-between;
   height: 5rem;
-  padding: 0 2rem;
+  //padding: 0 2rem;
```
Comment (LorenzoValentine) [inline src/components/NavBar.tsx:16]: Remove this comment

File: src/components/NavBar.tsx [hunk 2]
```diff
@@ -66,45 +72,114 @@ const NavButton = styled(Button)`
...
         <Logo
           src="/physics.svg"
           alt="Physics in the Browser Logo "
           width={50}
           height={50}
         />
-      </NavAnchor>
-      <Name>CFLowSim</Name>
-      <Nav>
+        <Name>CFLowSim</Name>
```
Comment (LorenzoValentine) [inline src/components/NavBar.tsx:125]: Change the `CFLowSim` to `CFlowSim`

**Comment by Lutra-Fs** [comment]
you cannot give your self suggestion 😢

File: src/components/NavBar.tsx (hunk 1 above)
Comment (Lutra-Fs) [inline src/components/NavBar.tsx:16]:
```suggestion
```

File: src/components/NavBar.tsx (hunk 2 above)
Comment (Lutra-Fs) [inline src/components/NavBar.tsx:125]:
```suggestion
        <Name>CFlowSim</Name>
```

**Review by Lutra-Fs — CHANGES_REQUESTED** [review]

**Comment by Lutra-Fs** [comment]
A quick question, control bar has not been touched, list it to another issue/pr?

**Review by Lutra-Fs — CHANGES_REQUESTED** [review]
Please resolve conflict in your local machine

**Review by Lutra-Fs — CHANGES_REQUESTED** [review]
Please resolve conflict in your local machine

**Pull request merged**